Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -4517,17 +4517,17 @@ private List<string> GetWhereClauses(InternalItemsQuery query, IStatement statem
                 }
             }
 
-            if (query.HasImdbId.HasValue)
+            if (query.HasImdbId.HasValue && query.HasImdbId.Value)
```

Comment:
An alternative is

```suggestion
            if (query.HasImdbId == true)
```

This may be slightly more optimized.